agent/taskresource/cgroup/cgroup.go

```diff
@@ -32,7 +33,7 @@ import (
 	"github.com/cihub/seelog"
 	"github.com/containerd/cgroups"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
-	"github.com/pkg/errors"
+	pkgerr "github.com/pkg/errors"
```

Comment:
I think we can remove this and only depend on the `"errors"` package. 
What say you?